src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.styles.ts

```diff
@@ -13,7 +13,12 @@ export const getStyles = (
     _props: IAdvancedSearchResultDetailsListStyleProps
 ): IAdvancedSearchResultDetailsListStyles => {
     return {
-        root: [classNames.root],
+        root: [
+            classNames.root,
+            {
+                marginTop: 0
```

Comment:
why does the root have a margin on top by default? Isn't it just a div?

Reply:
stack props, nice catch